Actions/IncrementVersionNumber/IncrementVersionNumber.ps1

```diff
@@ -30,20 +30,18 @@ try {
 
     $settings = $env:Settings | ConvertFrom-Json
 
-    # Ensure the repoVersion setting exists in the repository settings. Defaults to 1.0 if it doesn't exist.
-    Set-VersionInSettingsFile -settingsFilePath (Join-Path $baseFolder $RepoSettingsFile) -settingName 'repoVersion' -newValue $settings.repoVersion -Force # $RepoSettingsFile is defined in AL-Go-Helper.ps1
-
-    # Change repoVersion in repository settings
-    Set-VersionInSettingsFile -settingsFilePath (Join-Path $baseFolder $RepoSettingsFile) -settingName 'repoVersion' -newValue $versionNumber
-
     $projectList = @(GetProjectsFromRepository -baseFolder $baseFolder -projectsFromSettings $settings.projects -selectProjects $projects)
```

Comment:
Can we not change the repoVersion in the repo settings if it already exists there?

Reply:
How about if (projects = '*') then change repoVersion in repo settings? If it already exists there already

Reply:
What is the problem in having repoVersion in all projects and remove it from repo settings?
We also have the version number in all app.json files, right?

Reply:
I think we can discuss this later - for now - moving the change to modify the project level settings file is what the action did before.

The earlier version also didn't modify repo settings.: https://github.com/BusinessCentralApps/buildorder/pull/166/files

The only reason that it works is that all project settings basically can be set in repo settings.

We should then decide whether we should change this - and have repoversion and projectversion or something else. Lets not change the functionality here.